packages/babel-helper-create-class-features-plugin/src/misc.js

```diff
@@ -85,6 +85,9 @@ export function injectInitialization(path, constructor, nodes, renamer) {
 export function extractComputedKeys(ref, path, computedPaths, file) {
   const declarations = [];
 
+  const nearestBlock = path.find(
+    parentPath => parentPath.isBlockStatement() || parentPath.isProgram(),
```

Comment:
Sometimes the variable needs to be inserted where a block doesn't exist. Do we have a test for this code?

```js
() => class { [k] = 2 }
```

Reply:
good catch, added a new test case here (https://github.com/babel/babel/pull/10029/files#diff-fe07c1ba4a5c933d73ca27c26a08737a)